modules/luci-base/htdocs/luci-static/resources/uci.js

```diff
@@ -919,6 +919,8 @@ return baseclass.extend(/** @lends LuCI.uci.prototype */ {
 							window.setTimeout(try_confirm, 250);
 						else
 							return Promise.reject(rv);
+					} else {
+						document.dispatchEvent(new CustomEvent('uci-applied'));
```

Comment:
There's a dispatchEvent in ui.js on the normal confirm/apply flow, but not on this one.